CHANGELOG.md

```diff
@@ -4,16 +4,22 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Listener.status.addresses now allows users to configure whether they prefer IP addresses or DNS hostnames ([#233]).
```

Comment:
```suggestion
- `Listener.spec.preferredAddressType` now allows users to configure whether they prefer IP addresses or DNS hostnames ([#233]).
```

Reply:
The configuration field is in `ListenerClass`. It should probably be something closer to "`Listener.status.addresses` can now be configured to..."

Reply:
9c2ce25586bacc779e0dfedd95d674b704ddf023

Reply:
works for me